backend/JavaWebService/pom.xml

```diff
@@ -189,6 +187,41 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.6</version>
+                <configuration>
+                    <excludes>
+                        <exclude>com/kainos/ea/WebServiceApplication.class</exclude>
+                        <exclude>com/kainos/ea/WebServiceConfiguration.class</exclude>
+                        <exclude>com/kainos/ea/db/*</exclude>
+                        <exclude>com/kainos/ea/models/*</exclude>
+                        <exclude>com/kainos/ea/services/*</exclude>
+                    </excludes>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>prepare-agent</id>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>report</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>report</goal>
+                        </goals>
+                        <configuration>
+                            <!-- Sets the path to the file which contains the execution data. -->
+                            <dataFile>target/jacoco.exec</dataFile>
+                            <!-- Sets the output directory for the code coverage report. -->
+                            <outputDirectory>target/my-reports</outputDirectory>
```

Comment:
I would call it something about code coverage, so it's obvious what to expect when browsing the target directory